### ISSUE TITLE
Unknown node: math

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#901]
When running sphinx-build through

```
$ make latexpdf
```

I ran into the following problem:

```
# Sphinx version: 1.1.3
# Python version: 3.2.2
# Docutils version: 0.8.1 release
# Jinja2 version: 2.6
Traceback (most recent call last):
  File "/usr/lib/python3.2/site-packages/sphinx/cmdline.py", line 189, in main
    app.build(force_all, filenames)
  File "/usr/lib/python3.2/site-packages/sphinx/application.py", line 204, in build
    self.builder.build_update()
  File "/usr/lib/python3.2/site-packages/sphinx/builders/__init__.py", line 191, in build_update
    self.build(['__all__'], to_build)
  File "/usr/lib/python3.2/site-packages/sphinx/builders/__init__.py", line 252, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/lib/python3.2/site-packages/sphinx/builders/latex.py", line 104, in write
    docwriter.write(doctree, destination)
  File "/usr/lib/python3.2/site-packages/docutils/writers/__init__.py", line 77, in write
    self.translate()
  File "/usr/lib/python3.2/site-packages/sphinx/writers/latex.py", line 94, in translate
    self.document.walkabout(visitor)
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 173, in walkabout
    if child.walkabout(visitor):
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 165, in walkabout
    visitor.dispatch_visit(self)
  File "/usr/lib/python3.2/site-packages/docutils/nodes.py", line 1604, in dispatch_visit
    return method(node)
  File "/usr/lib/python3.2/site-packages/sphinx/writers/latex.py", line 1511, in unknown_visit
    raise NotImplementedError('Unknown node: ' + node.__class__.__name__)
NotImplementedError: Unknown node: math
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/901
- Originally reported by: Anonymous
- Originally created at: 2012-03-19T10:28:27.211

**Comment by shimizukawa** [comment]
_From Martin Ortbauer on 2013-02-19 20:59:05+00:00_

Hey I just stumbled upon the same error, and in my case it is because I hadn't
activated the mathjax extension. This is a bug in my opinion since matjax isn't needed by latex but it probaly has some side effects. As a workaround just activate it by adding:

extensions = ['sphinx.ext.mathjax']

to your conf.py

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-11 08:49:52+00:00_

Closes #901: Emit a warning when using docutils' new "math" markup without a Sphinx math extension active.

→ <<cset 2a663ee71c6dc91baab41c438135f4207cb6eaf1>>
